appinfo/routes.php

```diff
@@ -31,11 +32,17 @@
 			'url' => '/config',
 			'verb' => 'GET'
 		],
+		// TODO: Remove POST in next API release
 		[
 			'name' => 'config#updateAppConfig',
 			'url' => '/config/update',
 			'verb' => 'POST'
 		],
+		[
+			'name' => 'config#updateAppConfig',
+			'url' => '/config/update',
+			'verb' => 'PATCH'
+		],
```

Comment:
```suggestion
		[
			'name' => 'config#updateAppConfig',
			'url' => '/config/update',
			'verb' => 'PATCH'
		],
```

I think we can safely remove this as this is not part of our public API